uber/site_sections/registration.py

```diff
@@ -22,8 +22,14 @@
 
 
 def pre_checkin_check(attendee, group):
-    if c.NUMBERED_BADGES and not attendee.badge_num:
-        return 'Badge number is required'
+    if c.NUMBERED_BADGES:
```

Comment:
No major changes here, code got moved from several lines down to fix a bug for events without numbered badges.